Formula/telnetd.rb

```diff
@@ -0,0 +1,48 @@
+class Telnetd < Formula
+  desc "TELNET server (built from macOS Sierra sources)"
+  homepage "https://opensource.apple.com/"
+  url "https://opensource.apple.com/tarballs/remote_cmds/remote_cmds-54.50.1.tar.gz"
+  sha256 "156ddec946c81af1cbbad5cc6e601135245f7300d134a239cda45ff5efd75930"
+
+  keg_only :provided_pre_high_sierra
+
+  depends_on :xcode => :build
+
+  resource "libtelnet" do
+    url "https://opensource.apple.com/tarballs/libtelnet/libtelnet-13.tar.gz"
+    sha256 "e7d203083c2d9fa363da4cc4b7377d4a18f8a6f569b9bcf58f97255941a2ebd1"
+  end
+
+  def install
+    resource("libtelnet").stage do
+      xcodebuild "SYMROOT=build"
+
+      libtelnet_dst = buildpath/"telnetd.tproj/build/Products"
+      libtelnet_dst.install "build/Release/libtelnet.a"
+      libtelnet_dst.install "build/Release/usr/local/include/libtelnet/"
+    end
+
+    system "make",
+           "-C", "telnetd.tproj",
+           "OBJROOT=build/Intermediates",
+           "SYMROOT=build/Products",
+           "DSTROOT=build/Archive",
+           "CFLAGS=$(CC_Flags) -isystembuild/Products/",
+           "LDFLAGS=$(LD_Flags) -Lbuild/Products/"
+
+    bin.install "telnetd.tproj/build/Products/telnetd"
```

Comment:
`bin` feels like a weird place for a daemon. I don't know. Homebrew has some mixed rules on this so don't move it unless @ilovezfs feels strongly.

Reply:
sbin is likely a better fit. does it need root to run?

Reply:
@ilovezfs Unfortunately, I've never *actually used* `telnetd`... I just saw that some people complained about it being gone in 10.13 so I added it here. I don't know if it needs root privileges or not.

Reply:
@ilovezfs @DomT4 Ok, so all I needed to do to start `telnetd` manually was to RTFM (from `man 8 telnetd`):

> The -debug option may be used to start up telnetd manually, instead of through inetd(8).  If started up this way, port may be specified to run telnetd on an alternate TCP port number.

If the port number that `telnetd` tries to bind to is below 1024 (which is the default) then we need root access. Though, we can pass a custom port number to `-debug` which eliminates the need for root privileges. So I'm now not sure whether this should be in `bin` or `sbin`.

Reply:
Smells like `sbin` to me, personally. Especially since we leave `sshd` there.

Reply:
@DomT4 @ilovezfs Alright, should I install both `tnftpd` and `telnetd` to `sbin`, then?

Reply:
ILZ gets to make the final call on that one. Personal opinion: Yes.